src/redisearch_api.c

```diff
@@ -33,6 +33,12 @@ IndexSpec* RediSearch_CreateIndex(const char* name, const RSIndexOptions* option
     spec->indexer = NewIndexer(spec);
   }
 
+  if (options->score || options->lang) {
```

Comment:
Isn't it better to get the score_default and lang_default out of the rule. There is no real meaning for rule when used from LLAPI...

Reply:
Since this requires a change to rdb, issue #2185 was opened to be done next time the rdb version is changed